printer_mods/hartk1213/Voron0_Hinged_Top_Hat/Readme.md

```diff
@@ -4,65 +4,92 @@ This is my new design for a fold away hinged top hat for the V0, i have included
+
+#### NOTE: This step uses M3x8 screws
+next is to add these 200mm extrusions to the top hinge assembly created in previous step, these also use m3x8 screws
 
 # Step 5
 ![building the frame from the back to the front](Images/Assembly/step5.png) 
-next attach the 2 80mm extrusions up against the top hinge piece
+
+#### NOTE: This step uses M3x8 screws
+attach the 2 80mm extrusions up against the top hinge assembly
 
 # Step 6 
 ![building the frame from the back to the front](Images/Assembly/step6.png) 
-these are the 170mm extrusions that go up against the 80mm extruisons
+
+#### NOTE: This step uses M3x8 screws
+attach the 170mm extrusions that go up against the 80mm extrusions
 
 # Step 7
 ![building the frame from the back to the front](Images/Assembly/step7.png) 
+
+#### NOTE: This step uses M3x8 screws
 now add the 92mm extrusions to the front of the frame, now these are 92mm long but the gap between the top and bottom extrusion is 95mm, you want the top of the extrusion to be flush and leave a little gap in the bottom of them which is for the alignment pins
 
 # Step 8
- ![here](Images/Assembly/step8.png) these are the alignment pins 
- 
+![here](Images/Assembly/step8.png) 
+
+#### NOTE: This step uses M3x8 screws
+these are the alignment pins, they are used to register in the holes on top of the front idlers on the V0 and they just attach to the bottom of the 92mm extrusions
+
 # Step 9 
- ![here](Images/Assembly/step9.png)attach the front top and bottom extrusion
- 
- # Step 10
-  ![here](Images/Assembly/step10.png) next step is to attach the bottom hinge and top hinge together using the middle hinge assembly you made in step 2
-  
-  # final step
-  ![here](Images/Assembly/FinalAssembly.png) 
-  Add the panels to the top hat 
-
-
-bottom hinge uses m3x12 to attach to the frame using an m3 nut that needs to be slid into the top extrusion
-top hing which also uses an m3x12 and m3 nut in 1 of the 3 holes to attach it the other 2 are m3x8
-the middle hinge (middle screw) is an m3x16
-the middle hinge (2 outer screws are m3x20)
-other than those the rest are going to be m3x8
+![here](Images/Assembly/step9.png)
+
+#### NOTE: This step uses M3x8 screws
+attach the 2 front extrusion and you should have the completed frame now
+
+# Step 10
+![here](Images/Assembly/step10.png)
+
+#### NOTE: This step uses M3x20 screws
+what i would do next is attach the bottom hinge and top hinge together using the middle hinge assembly you made in step 2 and check to make sure every thing is moving correctly and lining up properly, and make any adjustments you see fit. 
+
+# final step
+![here](Images/Assembly/FinalAssembly.png) 
+
+Once the frame is moving correctly and everythins is working well together, add the panels to the top hat to enclose it completely 
+there are 2 panels that go in the back the inner one can be left out, i originally didnt have the final rear printed panel but now that its there the inner rear panel is no longer needed 
```

Comment:
`I` should be capitalized.  `didn't` needs an apostrophe.  `its there` should be `it's there`